vertical-pod-autoscaler/pkg/utils/status/status_object.go

```diff
@@ -124,7 +124,8 @@ func IsStatusValid(status *apicoordinationv1.Lease, leaseTimeout time.Duration)
 
 func isStatusValid(status *apicoordinationv1.Lease, leaseTimeout time.Duration, now time.Time) bool {
 	return status.CreationTimestamp.Add(leaseTimeout).After(now) ||
```

Comment:
CreationTimestamp is not optional I presume?

Reply:
AFIAK it is not.